test/library/standard/Types/copyable-records.bad

```diff
@@ -0,0 +1,8 @@
+copyable-records.chpl:78: error: Cannot default-initialize z: owned C
+note: non-nil class types do not support default initialization
+note: Consider using the type owned C? instead
+copyable-records.chpl:107: In function 'main':
+copyable-records.chpl:115: error: unresolved call 'CustomContainingNonNilableOwned.init(owned C)'
+copyable-records.chpl:95: note: this candidate did not match: CustomContainingNonNilableOwned.init()
```

Comment:
Does this test only track one failing style or two?  I'd like it to be split into parts that work, and individual parts that fail, since it seems like there is more than one thing tracked by this test that isn't tracked by the other test

Reply:
I split it into 3 tests and made the future focused on the case that isn't working.